crates/aranya-daemon/src/policy.md

```diff
@@ -111,13 +104,45 @@ struct KeyIds {
     sign_key_id id,
     enc_key_id id,
 }
+
+// Defines a device on the team.
+struct DeviceInfo {
+    // ID of the device.
+    device_id id,
+    // Device precedence.
+    precedence int,
+    // Signing key id.
+    sign_key_id id,
+    // Encryption key id.
+    enc_key_id id,
+}
+
+// Defines a label on the team.
+struct LabelInfo {
+    // ID of the label.
+    label_id id,
+    // Name of the label.
+    name string,
+    // ID of device that created the label.
+    author_id id,
+}
+
+// Defines a role on the team.
+struct RoleInfo {
+    // ID of the role.
+    role_id id,
+    // Name of the role.
+    name string,
+    // ID of device that created the role.
+    author_id id,
+}
```

Comment:
Only actions and effects are part of the policy's stable API. Using structs inside an effect seems like a violation of that.